README.md

```diff
@@ -1,20 +1,19 @@
 # Running Kubernetes Anywhere
 
-This project uses Weave to dramatically simplify Kubernetes deployment - anywhere. It is by far the easiest way to get started on a single machine, and later scale-out to any infrastructure seamlessly. We use Weave Net as a cluster management network. This enables complete portability, and for example allows one to move or clone the entire cluster. Even TLS setup is fully transparent.
+Kubernetes Anywhere uses Weave Net to dramatically simplify Kubernetes deployment on to any cloud provider. It is by far the easiest way to get started on a single machine and to subsequently scale-out to any infrastructure. By implementing Weave Net as a cluster management network, complete portability is ensured, allowing you to move or clone an entire cluster. Even setting up Transport Layer Security (TLS) is fully transparent.
```

Comment:
Let's say "bootstrap/management" instead of "management", someone pointed out that word bootstrap explains it a lot better.